src/app/content/studyGuides/reducer.spec.ts

```diff
@@ -34,26 +74,6 @@ describe('study guides reducer', () => {
     expect(state.isEnabled).toEqual(true);
   });
 
-  it('sets summary filters', () => {
-    const state = reducer({
-      ...initialState,
-      summary: {
-        ...initialState.summary,
-        filters: {
-          colors: [],
-          default: false,
-          locationIds: [],
-        },
-      },
-    }, actions.setSummaryFilters({
-      locationIds: ['id'],
-    }));
-
-    expect(state.summary.filters.locationIds[0]).toEqual('id');
-    expect(state.summary.filters.locationIds.length).toEqual(1);
-    expect(state.summary.loading).toEqual(true);
-  });
-
```

Comment:
@TomWoodward removed this because it's now unnecessary (as you pointed out) but am I correct that any test that uses this to set the filters will now need to render the app in order to set them via locationChange?

Reply:
what is the difference between setSummaryFilters and updateSummaryFilters.

in response to the user changing the selected filters, you can dispatch locationchange directly or via a helper, thats what i'm referring to in my comment. there might be complicating factors though idk